core/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -453,7 +456,7 @@ private void removeIndex(Index index, String reason, boolean delete) {
                 final IndexSettings indexSettings = indexService.getIndexSettings();
                 listener.afterIndexDeleted(indexService.index(), indexSettings.getSettings());
                 // now we are done - try to wipe data on disk if possible
-                deleteIndexStore(reason, indexService.index(), indexSettings, false);
+                deleteIndexStore(reason, indexService.index(), indexSettings, true);
```

Comment:
I'm worried about this change (I know I did it). I would live a sanity check with @dakrone that this was a bug (though it didn't really have an effect due to how we were using this method). @dakrone I'll reach out over voice to explain